packages/cozy-harvest-lib/src/components/Popup.jsx

```diff
@@ -0,0 +1,143 @@
+import { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * customized function to center a popup window
+ * @param  {string} url
+ * @param  {string} title
+ * @param  {string|number} w
+ * @param  {string|number} h
+ * @return {Window}       Popup window
+ */
+// source https://stackoverflow.com/a/16861050
+export function popupCenter(url, title, w, h) {
+  /* global screen */
+  // Fixes dual-screen position
+  //                      Most browsers      Firefox
+  var dualScreenLeft = window.screenLeft || screen.left
+  var dualScreenTop = window.screenTop || screen.top
+
+  var width = window.innerWidth
+    ? window.innerWidth
+    : document.documentElement.clientWidth
+    ? document.documentElement.clientWidth
+    : screen.width
+  var height = window.innerHeight
+    ? window.innerHeight
+    : document.documentElement.clientHeight
+    ? document.documentElement.clientHeight
+    : screen.height
+
+  var left = width / 2 - w / 2 + dualScreenLeft
+  var top = height / 2 - h / 2 + dualScreenTop
+  // need to be set here to get from the OAuth opener
+  var newWindow = window.open(
+    '',
+    title,
+    `scrollbars=yes, width=${w}, height=${h}, top=${top}, left=${left}`
+  )
+  newWindow.location.href = url
```

Comment:
Is there a good reason to no open the URL at first? Within an `inAppBrowser` you can't set the URL afterwards

Reply:
I think it's a way to ensure that the popup will be opened and not be blocked by browser. I need to find a reference about this technic and will add a comment.

Reply:
Update: I was not able to find anything about this technic. So let's keep the code where it's removed.